src/App.js

```diff
@@ -14,11 +14,75 @@ const goodsFromServer = [
   'Garlic',
 ];
 
-const App = () => (
-  <div className="App">
-    <h1>Selected good: -</h1>
-    {goodsFromServer.length}
-  </div>
-);
+class App extends React.Component {
+  state = {
+    selectedGoods: [],
+  }
+
+  select = (e) => {
+    const { selectedGoods } = this.state;
+    const selectedElement = e.target.parentElement;
+
+    selectedElement.classList.add('selected');
+    selectedGoods.push(selectedElement.innerText.split(' ')[0]);
+
```

Comment:
don't use target and DOM, if you want to change class, then add condition into className attribute, if you need any kind of data store them into the state, and use it from the state.